eng/Version.Details.xml

```diff
@@ -1,6 +1,19 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Dependencies>
-  <ProductDependencies></ProductDependencies>
+  <ProductDependencies>
+    <Dependency Name="Microsoft.NETCore.Runtime.CoreCLR" Version="3.0.0-preview-27218-02">
+      <Uri>https://github.com/dotnet/coreclr</Uri>
+      <Sha>6f4c9d6427829807ad1fc4432f6f7d544e982c67</Sha>
+    </Dependency>
+    <Dependency Name="Microsoft.NETCore.ILAsm" Version="3.0.0-preview-27218-02">
+      <Uri>https://github.com/dotnet/coreclr</Uri>
+      <Sha>6f4c9d6427829807ad1fc4432f6f7d544e982c67</Sha>
+    </Dependency>
+    <Dependency Name="Microsoft.NET.Sdk.IL" Version="3.0.0-preview-27218-02">
```

Comment:
>Microsoft.NET.Sdk.IL [](start = 22, length = 20)

This is not defined in `Versions.props` so it won't be updated there hence you won't use the latest version

Reply:
I believe we don't need it there since it's already in global.json.

Reply:
Right! Just checked `global.json` :ok: 

---
In reply to: [243077447](https://github.com/dotnet/corefx/pull/34162#discussion_r243077447) [](ancestors = 243077447)